.github/workflows/test.yml

```diff
@@ -27,6 +29,13 @@ jobs:
         toolchain: ${{ matrix.rust }}
         override: true
 
+    - name: Set up MSYS2 and install clang
+      if: matrix.os == 'windows-latest'
+      uses: msys2/setup-msys2@v2
+      with:
+        update: true
+        install: mingw-w64-x86_64-clang
+
```

Comment:
is this also related to this issue?

Reply:
Not really, but the pipeline is now failing because it can't compile RocksDB